README.md

```diff
@@ -1,4 +1,4 @@
-Ballerina Uuid Package
+Ballerina Uuid Library
```

Comment:
I think this should be UUID, right? @MadhukaHarith92

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-uuid/pull/363